proprietary/tokens/README.md

```diff
@@ -1,3 +1,141 @@
+
+We use these tokens in our [CSS components](https://www.npmjs.com/package/@amsterdam/design-system-css) and [React components](https://www.npmjs.com/package/@amsterdam/design-system-react).
+Any web-based application for Amsterdam can implement the design system through this package.
+
+## Install the package
+
+Use this package alone if you want or need to use the design tokens directly in your project.
+Note that our [CSS components](https://www.npmjs.com/package/@amsterdam/design-system-css) and [React components](https://www.npmjs.com/package/@amsterdam/design-system-react) provide more functionality and apply the tokens automatically.
+
+```sh
+pnpm add @amsterdam/design-system-tokens
+```
+
+## Import the tokens
+
+Find the tokens in the `dist` directory and import them into your project.
+
+Tokens are typically applied as custom properties in CSS.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+```
+
+### Root or class selector
+
+The stylesheets wrap the tokens into a `:root` selector.
+If that doesn’t work for your project, use `index.theme.css` instead, which uses an `.ams-theme` selector.
+Add that class to a root element of your application.
+
+### Compact mode
+
+Applications need less whitespace and smaller text sizes than websites.
+Add the compact stylesheet to override the values of these tokens.
+
+```ts
+import "@amsterdam/design-system-tokens/dist/index.css";
+import "@amsterdam/design-system-tokens/dist/compact.css";
+```
+
+Dark mode will work the same way when it’s ready.
+
+### Other formats
+
+The tokens can be imported as Sass variables as well.
+
+```sass
+@import "@amsterdam/design-system-tokens/dist/index.scss"
+```
+
+To use the tokens in JavaScript, import the JSON file.
+
+```ts
+import tokens from "@amsterdam/design-system-tokens/dist/index.json";
+```
+
+## Use the tokens
+
+All tokens' names start with a prefix of `ams-`.
+The name of component tokens ends with the CSS property that uses them, e.g. `-font-size`.
+
+### Component tokens
+
+Every component uses tokens for every property that expresses branding.
+Apply these values in your stylesheets to the corresponding components.
+
+```html
+<button class="my-button" type="button">Button label</button>
+```
+
+```css
+.my-button {
+  font-family: var(--ams-button-font-family);
+  background-color: var(--ams-button-primary-background-color);
+}
+```
+
+### Common tokens
+
+Various groups of components share visual design characteristics.
+E.g. all kinds of links have the same colour, and the borders of various inputs are equally thick.
+We use common tokens to express these relations.
+This also makes it easier to change this shared appearance in the future.
+
+Prefer using common tokens over component tokens.
+If
+
+```html
+<input class="my-input" type="text" />
+```
+
+```css
+.my-input {
+  font-size: var(--ams-common-input-font-size);
+  border-width: var(--ams-common-input-border-width);
```

Comment:
These are none-existent tokens, no? I would avoid that, to prevent copy paste frustration.

Reply:
I know but they do illustrate the example better than other token names. And I intend to adopt this naming strategy for brand and common tokens. I’ve added a notice that these don’t exist yet.